### ISSUE TITLE
Priority Queue In Java

### ISSUE THREAD

**Issue by JainStuti25** [FOSS-UCSC/FOSSALGO#556]
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS)

-   [ x ] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [ x] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [ x ] Assign this issue to me (I want to work on this)

   -   [ x ] I ensure that I am not already assigned to 2 or more issues
  
-   [ x] I will send my PR only after I'm assigned to this issue by any maintainer

**Comment by RavinduSachintha** [comment]
@JainStuti25 good luck! read the contributing doc before starting. :+1:

**Comment by RavinduSachintha** [comment]
@JainStuti25 sorry. I have mistaken. the priority queue is not a defined data structure in the system yet. we have restricted the folder creation privileges for contributors. If you want to work on this issue, I can create a folder for that. if not we can assign you to another issue that is related to a already defined data-structure/algorithm.

**Comment by JainStuti25** [comment]
@RavinduSachintha It would be great if you can create a folder for the Priority Queue. I would also like to work on other issue as well.

**Comment by Danushka96** [comment]
Hi @JainStuti25 
The folder is created, fix the compile issues in the #558
